Formula/ltex-ls.rb

```diff
@@ -0,0 +1,41 @@
+class LtexLs < Formula
+  desc "LSP for LanguageTool with support for Latex, Markdown and Others"
+  homepage "https://valentjn.github.io/ltex/"
+  url "https://github.com/valentjn/ltex-ls/archive/refs/tags/15.2.0.tar.gz"
+  sha256 "59209730cb9cda57756a5d52c6af459f026ca72c63488dee3cfd232e4cfbf70a"
+  license "MPL-2.0"
+  head "https://github.com/valentjn/ltex-ls.git", branch: "develop"
+
+  depends_on "maven" => :build
+  depends_on "python@3.9" => :build
```

Comment:
can this use `python@3.10`?
```suggestion
  depends_on "python@3.10" => :build
```